### ISSUE TITLE
Calibration upload f0cc69

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#1117]
Calibration images were uploaded to the directory that starts with “`f0cc69_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

Additional comments by the uploader:

> Hallo, ich kann kein Korrekturprofil für mein Olympus Zuiko ED 8mm F3.5 Fisheye Objektiv finden. Ich habe ein paar Bilder mit meiner E-620 mit diesem Objektiv gemacht. Sind diese ausreichend, um die Korrektur mit dieser Linse zu eröglichen? Ich kann auch gern weitere Bilder machen, sollte ihr Service noch nutzbar sein.

**Comment by origpumu** [comment]
Hi, I'm the uploader. Sorry for german comment.

**Comment by uchrisu** [comment]
I'm wondering if a correction of the distortion of a lens makes sense? 

I can try to do TCA correction on the last 3 pictures. They are Iso 100 and have the highest contrast.

What do the other think?

**Comment by origpumu** [comment]
> I'm wondering if a correction of the distortion of a lens makes sense?
Why not to correct the distortion of this lens? Is there a technical reason? Are the pictures not suitable to get the correction informations?
All other fisheye lenses (https://wilson.bronger.org/lensfun_coverage.html) have distortion correction in its profiles. I can take another shots for you, if you want them.

Thank you for your time.

**Comment by uchrisu** [comment]
Okay, thanks for the answer. I was not sure how other fish-eyes were handled. I will do the job this week and let you know the results. Thank you very much for the pictures, I think there are some of them that will do.

**Comment by junkyardsparkle** [comment]
If using "normal" methods used for rectilinear lenses isn't working out (it can be really hard to get good images for this with a fisheye) an alternate method is to take a series of shots for stitching a 360 horizontal panorama in hugin, ideally from a tripod, with generous overlap between shots.

**Comment by uchrisu** [comment]
I do not get it done with good results with the hugin workflow at the moment. Maybe someone else who is more familiar with the correction of fisheyes can help. Otherwise I will try it later again.

**Comment by junkyardsparkle** [comment]
I downloaded the files (only the f/4.5 ones) and selected P3094578.ORF one as having a pretty good line touching the top edge of the frame and *almost* spanning completely to both sides. Making sure to set the lens type to "Full frame fisheye" in hugin, I added control points like this:

![screenshot-1584042585](https://user-images.githubusercontent.com/4522189/76563123-057ff300-6464-11ea-8c4a-43e6280deca4.jpg)

Running the optimizer with only "b" parameter selected gave a very good fit (correction was very small - remember that this is correcting only to ideal fisheye, not de-fishing at this stage), so I left it at that, and used the same image for TCA correction. The resulting profile gives good results on the other images in darktable. If you don't feel like making another attempt I'll upload it here.

@origpumu If you don't want this screenshot displayed here, let me know and I'll remove it... otherwise, it's a very good example picture to keep here for instructional purposes.

**Comment by origpumu** [comment]
Feel free to use the picture as you want. Thatswhy I've taken the picture and uploaded it. 

Thank you for your time!

**Comment by uchrisu** [comment]
I took ...3094568.ORF since there is a line really at the edge and going over the full width. 
I also got a possibly good correction so somehow more straight lines but still in the fisheye look.

That was the intention of my initial question. Should we try a rectilinear correction of this lens?

**Comment by uchrisu** [comment]
Okay, marking it as fisheye and then taking the transformation to rectilinear in e.g. darktable works quite good. I tried (a,b,c) fitting an git these results:
![Bildschirmfoto vom 2020-03-13 09-28-46](https://user-images.githubusercontent.com/9560869/76603619-9068f800-650d-11ea-92c0-8c93261c3987.png)

Not quite perfect, but much better then before. 
@origpumu Can you post your rectilinear results for one picture for comparison? Maybe we take the single parameter (a) correction then, if it delivers the better results.

**Comment by junkyardsparkle** [comment]
Nice. Here's my result using only poly3 ("b" parameter) correction. It seemed to be sufficient, but if the more complex correction produces better results, it's worth using it. Sorry, maybe I should have made more clear the distinction between "correction" and "projection remapping", but it looks like you figured it out. 
[Zuiko_8mm_fisheye.xml.txt](https://github.com/lensfun/lensfun/files/4330947/Zuiko_8mm_fisheye.xml.txt)

**Comment by uchrisu** [comment]
I think both are not perfect. The one-parameter of @junkyardsparkle left a bit of barrel-like distortion while my parameters leave a bit of pincushion distortion. I will attempt a new try, maybe it is possible to get better results.

**Comment by junkyardsparkle** [comment]
Yes, and taking a closer look using my correction combined with re-projecting to rectilinear, I now think probably the more complex correction is needed; the remaining "barrel-like" distortion with mine has a little bit of "mustache" curvature as well. A really good calibration may require stitching a panorama, though.
